testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
K, I'll combine them, then. 

On this subject, the base reader/writer test classes in coordinates.base don't have a tearDown/close and so looks like the tests end with a bunch of files still open at the end - I assume that's a bad thing?